tech-committee-charter.md

```diff
@@ -61,6 +61,30 @@ The development process will include a process for the TC to oversee and approve
 
 The TC and entire technical community will follow any processes as may be specified by the Cloud Native Computing Foundation relating to the intake and license compliance review of contributions, including the CNCF IP Policy.
 
+### Sponsorship Requirements
+
+The TC will establish and maintain sponsorship across the SIGs and subprojects of OpenTelemetry.
+
+SIGs will be classified into three classifications of Sponsorship requirement:
+
+* *Escalation Sponsorship*: The TC sponsor will be a primary means of escalation of cross-project concerns from the SIG. This communication is primarily
+  offline, e.g. via CNCF slack or issue communication. The TC sponsor will check-in with SIG leadership quarterly.  Additionally, the TC will aide the SIG
+  in developing quarterly road maps during this check-in.
+* *Guiding Sponsorship*: The TC sponsor is an active participant of the SIG. The sponsor will help direct the SIGs efforts to match the technical goals of OpenTelemetry.
+* *Leading Sponsorship*: The TC sponsor is an active member of SIG leadership, driving the direct success of the project or proposal.
+
+As part of their active leadership, every TC member will be required to sponsor SIGs across OpenTelemetry.
+
+Every TC member is required to provide a minimum of one (1) *Escalation* sponsorship. All SIGs requiring escalation sponsorship will be divided across the TC.
+
+Every TC member is required to provide a minimum of two (2) *Guiding* or *Leading* sponsorships.
+
+Every TC member is limited to a maximum of two (2) *Leading* sponsorships. This is to keep OpenTelemetry efforts focused and reduce fragmented attention on
+core issues.
+
+A TC member may have a time where they are not meeting the minimum sponsorship requirement, e.g. when major efforts have acheived results, and new projects are being formulated.
```

Comment:
May also want to consider a "time off policy" for extended vacations and such. Another TC member presumably would be expected to cover duties for some period of time?